pkg/manifestclient/list.go

```diff
@@ -240,7 +240,10 @@ func allPossibleListFileLocations(sourceFS fs.FS, requestInfo *apirequest.Reques
 		allPossibleListFileLocations = append(allPossibleListFileLocations, filepath.Join(clusterParts...))
 
 		namespaces, err := allNamespacesWithData(sourceFS)
-		if err != nil {
+		switch {
+		case errors.Is(err, fs.ErrNotExist):
+			return allPossibleListFileLocations, nil
+		case err != nil:
```

Comment:
Makes sense to me. So IIRC this branch takes all possible file locations for all resources, namespaced or not.
Previously, we scraped `cluster-scoped-resources` and then tried to scrape all existing namespace directories. If the namespace directories didn’t exist, it would throw an error and stop. Now, it just returns all the file locations it knows about so far without failing.